airflow/providers/elasticsearch/BACKPORT_PROVIDER_CHANGES_2020.11.07.md

```diff
@@ -0,0 +1,9 @@
+
+
+### Release 2020.11.7
```

Comment:
Can you remind me again please on why we strip `0` in the date -- I can vaguely remember it had soemthing to do with PyPI release convention / parsing or stripping of 0??

Reply:
Yep. That's it. PyPI won't accept 2020.11.07 as version.

Reply:
And we are releasing 11.13 eventually so no issue whatsoever :)